### ISSUE TITLE
No module named "farm.conversion"

### ISSUE THREAD

**Issue by guggio** [deepset-ai/FARM#358]
**Describe the bug**
Imports fail with a ModuleNotFoundError as in the stack trace below. 

**Error message**
```
<ipython-input-3-4145fb20b75c> in <module>()
      6 from farm.modeling.language_model import Bert
      7 from farm.modeling.prediction_head import TextClassificationHead
----> 8 from farm.modeling.adaptive_model import AdaptiveModel
      9 from farm.modeling.optimization import initialize_optimizer
     10 from farm.train import Trainer

/usr/local/lib/python3.6/dist-packages/farm/modeling/adaptive_model.py in <module>()
     12 from transformers.modeling_auto import AutoModelForQuestionAnswering, AutoModelForSequenceClassification, AutoModelForTokenClassification, AutoModelWithLMHead
     13 
---> 14 from farm.conversion.onnx_optimization.bert_model_optimization import main as optimize_onnx_model
     15 from farm.data_handler.data_silo import DataSilo
     16 from farm.data_handler.processor import SquadProcessor

ModuleNotFoundError: No module named 'farm.conversion'
```
**To Reproduce**
Try to import the AdaptiveModel.

**System:**
 - OS: macOS Catalina 10.15.4 running on Colab
 - FARM version: 0.4.3

**Comment by tanaysoni** [comment]
Hi @guggio, are you running on Google Colab or your local machine?

If you're running locally, can you try uninstalling farm with `pip uninstall farm` and re-installing from the latest master branch with `pip install -e .` from the FARM root directory?

**Comment by guggio** [comment]
Hi @tanaysoni I was running on google colab. I just retried it with two different approaches to install Farm on Colab:
First:
```
!git clone https://github.com/deepset-ai/FARM.git
!pip install -r FARM/requirements.txt
!pip install FARM/
```
This used to work, but now fails with the mentioned exception.

Second:
`!pip install farm==0.4.3`
It is the first time I used this way to install it and it works fine. Do you have an explanation, why this approach works and the other fails?

**Comment by aloizel** [comment]
@tanaysoni add `pip install -e .` seems to correct the problem thanks

**Comment by tanaysoni** [comment]
Hi @guggio, in the first case, FARM is installed from the latest master branch. It was breaking due to missing `__init__.py` files, which is now fixed in #365.

If you want to try out cutting-edge(yet to be released) features, then use approach 1. For working with a stable release, use approach 2.
